pkg/profile/helm.go

```diff
@@ -81,3 +95,26 @@ func (p *Profile) makeHelmChartSpec(chart string, version string) helmv2.HelmCha
 		Version: version,
 	}
 }
+
+func (p *Profile) makeDefaultValuesCfgMap(name, data string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
```

Comment:
anyone know if these values exist somewhere I can use?

Reply:
I'm not sure I understand the question, sorry.... What do you mean exists somewhere..?

Reply:
so like for other types we use constants, eg:
```go
			Kind:       sourcev1.HelmRepositoryKind,
			APIVersion: sourcev1.GroupVersion.String(),
```

Reply:
Oh, I see! It's `metav1` :)

Reply:
`	v1 "k8s.io/api/core/v1"`

Reply:
hmmm no it has to be a string. i don;t think it exists?

Reply:
Oh yes! You're right. Yeah, the core ones don't have Kind defined as a string.